server.js

```diff
@@ -143,6 +143,15 @@ app.post('/api/webhooks/invoice/updated', (req, res) => {
         res.send('payment not ready to allocate')
     }
 })
+app.post('/api/webhooks/invoice/delete', async (req, res) => {
+    const invoiceId = req.body.data.object.id
+    try {
+        await apiModules.automations.deleteDraftInvoicesFromStripe({ invoiceId })
```

Comment:
Refactor this to call the function in a new module called `/api/modules/budgeting.js`

- sort index.js A-Z with new module
- Move and rename `deleteDraftInvoicesFromStripe` to `deletePaymentByStripeInvoiceId`